### PR TITLE
[OCE] Dependency updates

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Update dependencies in UI lib as part of OCE rotation
-adds new colours to v9 theming shim
# Why

Keeps us up to date

https://skype.visualstudio.com/SPOOL/_workitems/edit/3628655
# How Tested

Built and tested locally

### CODE REVIEW

File: packages/react-components/src/theming/v9ThemeShim.ts [hunk 1]
```diff
@@ -19,6 +19,13 @@ import { blackAlpha, whiteAlpha, grey, grey10Alpha, grey12Alpha } from './themeD
  */
 const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
   return {
+    colorBrandBackground3Static: palette.themeLighter,
```
Comment (JamesBurnside) [inline packages/react-components/src/theming/v9ThemeShim.ts:22]: was this from the types update?

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/theming/v9ThemeShim.ts (hunk 1 above)
Comment (dmceachernmsft) [reply packages/react-components/src/theming/v9ThemeShim.ts:22]: yes they added new properties to this type. I am not sure these map correctly, I am not sure what the process was to make that decision before

Comment (emlynmac) [reply packages/react-components/src/theming/v9ThemeShim.ts:22]: This should be fine....

Comment (vhuseinova-msft) [reply packages/react-components/src/theming/v9ThemeShim.ts:22]: Adding colors is alright, it's a reccomended way to do this, [when we have v8 and v9 themes in the same app.](https://react.fluentui.dev/iframe.html?viewMode=docs&id=concepts-migration-from-v8-components-theme-migration--page#creating-compatible-themes)(this link also attached above this const)
The only thing, the colours should be taken from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts (see the link above for more details)

**Review by JoshuaLai — APPROVED** [review]

**Pull request merged**